pkgs/top-level/all-packages.nix

```diff
@@ -712,7 +712,9 @@ in
 
   azure-cli = callPackage ../tools/admin/azure-cli { python = python3; };
 
-  azure-storage-azcopy = callPackage ../development/tools/azcopy { };
+  azure-storage-azcopy = callPackage ../development/tools/azcopy {
+    buildGoModule = buildGo112Module;
```

Comment:
I've merged https://github.com/NixOS/nixpkgs/pull/74811 so this should not be necessary